art/test2.py

```diff
@@ -255,12 +285,21 @@
 knight:
 ṮḕṠṮ
 <BLANKLINE>
+love1:
+ƬƐSƬ
+<BLANKLINE>
 mirror:
 ɈƨǝɈ
 <BLANKLINE>
 mirror_flip:
 ʇsǝʇ
 <BLANKLINE>
+rev1:
```

Comment:
I suggest to remove these 2 fonts.
They are not clear.

Reply:
OK, I'll do that.